Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,463 @@
+Remote Desktop File. Then click "Get Password" and paste in the private key from the keypair you created previously.
+
+At this point you can connect to the Windows server using RDP.
+
+> Here you can setup your server to run and listen on a certain port on this machine, our example server runs on port 7777.
+
+### Run the Server Sidecar
+
+The server sidecar enables Catena to be aware of your game server without requiring any changes to the code of the game server itself. It is additionally able to make requests on behalf of your game server
+if you so choose. See [Sidecar](Sidecar.md).
+
+First download the Sidecar to the machine, the latest release can be found [here](https://github.com/CatenaTools/sidecar/releases/tag/v0.0.1). Alternatively, clone and build the sidecar.
+
+In the same directory as the sidecar, add a `config.json.` There is an example in the  sidecar repo here: https://github.com/CatenaTools/sidecar/blob/main/config.json
+
+```json
+{
+  "motd": "Hello Catena",
+  "app": {
+    "logging": {
+      "level": "debug",
+      "encoder": "console"
+    },
+    "event_source": {
+      "use": "virtual_server_event_source",
+      "configs": {
+        "catena_api_event_source": {
+          "GrpcListenAddr": "0.0.0.0:8080"
+        },
+        "virtual_server_event_source": {
+          "strategy": "continuous_strategy",
+          "continuous_strategy": {
+            "RequestInterval": "5s"
+          }
+        }
+      }
+    },
+    "resolver": {
+      "use": "ec2_resolver",
+      "configs": {
+        "ec2_resolver": {
+          "MetadataBaseUrl": "http://169.254.169.254",
+          "AvailablePorts": [
+            {
+              "Min": 7777,
+              "Max": 7777
+            }
+          ]
+        },
+        "static_resolver": {
+          "Address": "server.local:8080"
+        }
+      }
+    },
+    "backend": {
+      "configs": {
+        "catena_backend": {
+          "url": "https://platform.dev.catenatools.com"
+        },
+        "noop_backend": {}
+      },
+      "use": "catena_backend"
+    },
+    "sidecar": {
+      "DebugPanelAddr": "localhost:8081",
+      "MatchManagementStrategy": {
+        "AutoPopulateIDs": true
+      }
+    }
+  }
+}
+```
+
+The key is to set the following fields:
+
+`app.backend.catena_backend.url` field, which must point to your deployed instance url.
+`app.resolver.configs.ec2_resolver.Min` - The min port to use for clients to connect.
+`app.resolver.configs.ec2_resolver.Max` - The max port to use for clients to connect.
+
+_For this demo, min and max port should be the same, and be the port your server runs on._
+
+After which you can launch the sidecar using `cmd.exe`
+
+At this point you should see the sidecar querying for matches repeatedly. that's it, Catena know's about your server and it is requesting matches We're done!
```

Comment:
This runs the sidecar, but the reader's DGS is not yet on the machine. What is the recommended deploy flow for them?